### DISCUSSION TITLE
Is "BAD PASSWORD:" a crucial part of the error message?

### DISCUSSION THREAD

**Discussion by snwh** [libpwquality/libpwquality#79] [Q&A]
Hi there, I've an inquiry from downstream. :) 

GNOME display manager uses the full text of error messages right from libpwquality when enforcing a password change for users (example below). I'm curious if appending `BAD PASSWORD:` to the message here is necessary? From the GNOME design perspective, that bit of the message is somewhat redundant in-context, so I'm curious if that could be removed. Doing so would clean up the string somewhat and give more room to use other cues such as applying error/warning styles to the message or text entry, or appending a warning icon, etc. to enforce the message "it's bad" instead.

![image](https://github.com/libpwquality/libpwquality/assets/1217306/4d949c73-cd07-4a69-809e-af23a5c16f2c)

**Comment by t8m** [comment]
IMO the `BAD PASSWORD:` prefix is clearly redundant.
